src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.hmcts.ccd.datastore.befta;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+import uk.gov.hmcts.befta.data.UserData;
+import uk.gov.hmcts.befta.exception.FunctionalTestException;
+
+public class DataStoreTestAutomationAdapter extends DefaultTestAutomationAdapter {
+
+    private Logger logger = LoggerFactory.getLogger(DataStoreTestAutomationAdapter.class);
```

Comment:
prefer @Sl4j if possible

Reply:
that is because @Sl4j was introduced only recently. But we should prefer that one now